mocks_test.go

```diff
@@ -6,7 +6,8 @@ import (
 	"time"
 	"unicode/utf8"
 
-	"github.com/go-kit/kit/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
```

Comment:
`dto "github.com/prometheus/client_model/go"` is only used for mocking prometheus metrics via our metric interfaces